velox/docs/develop/operators.rst

```diff
@@ -665,9 +667,11 @@ number starting with 1.
    * - unnestVariables
      - Input columns of type array or map to expand.
    * - unnestNames
-     - Names to use for expanded columns. One name per array column. Two names per map column.
+     - Names to use for expanded columns. One name per array column. Two names per map column. If array of rows (and not legacyUnnest), then one column per child type of the row.
    * - ordinalityName
      - Optional name for the ordinality column.
+   * - legacyUnnest
+     - Array of rows are typically expanded to multiple columns, one for each child type of the row. However, with legacyUnnest, array of rows (like other array types) are expanded to a single column of the row type.
```

Comment:
perhaps, legacyUnnest -> flattenArrayOfRow or unnestArrayOfRow

Reply:
@mbasmanova : legacyUnnest unnests array of rows as a single column of row type. (!legacyUnnest) unnests as multiple columns of the child row types. So I presume you are suggesting flattenArrayOfRow or unnestArrayOfRow for the latter behavior (!legacyUnnest) (Your comment says replace legacyUnnest with those)